app/views/admin/instances/show.html.haml

```diff
@@ -84,5 +84,5 @@
       = link_to t('admin.instances.delivery.stop'), stop_delivery_admin_instance_path(@instance), data: { confirm: t('admin.accounts.are_you_sure'), method: :post }, class: 'button'
     - else
       = link_to t('admin.instances.delivery.restart'), restart_delivery_admin_instance_path(@instance), data: { confirm: t('admin.accounts.are_you_sure'), method: :post }, class: 'button'
-    - unless @instance.delivery_failure_tracker.available? && @instance.accounts_count > 0
+    - unless @instance.delivery_failure_tracker.available? && @instance.accounts_count > 0 && ! @instance.domain_block&.suspend?
```

Comment:
I think it would be clearer written as `if !@instance.delivery_failure_tracker.available? || @instance.accounts_count.zero? || @instance.domain_block&.suspend?`